src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1979,11 +1980,18 @@ public void ImportLibrary(object parameter)
             DialogResult result = openFileDialog.ShowDialog();
             if (result == DialogResult.OK)
             {
-                foreach (var file in openFileDialog.FileNames)
+                try
```

Comment:
great. can you write a Test around this? It should be tested against UI. May be you can specify any dummy dll and call Import Library. Check whether the exception is caught correctly.